### ISSUE TITLE
cURL and wp_remote_get() for graph.instagram.com: Segmentation error

### ISSUE THREAD

**Issue by pabamato** [10up/wp-local-docker-v2#271]
**Describe the bug**

calling `wp_remote_get( 'https://graph.instagram.com/' ) ` ends in a blank page (no HTML in source)
 
**Steps to Reproduce**

1. 10updocker shell
2.run `curl --location --request GET 'https://graph.instagram.com/'`
3. You will get a `Segmentation fault` message

**Expected behavior**

cURL should return the body from that URL (https://graph.instagram.com/):
"Sorry, this content isn't available right now"

**Screenshots**

**Environment information**
 - Device: MacBook Pro M1
 - OS: macOS 12.1 (21C52)

 - WordPress version: 5.8.1 / 5.8.2

**Additional context**
Found the following:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=989604
https://github.com/docker/for-mac/issues/5846
Adding --tls-max 1.2 worked for me on the command line inside the php-fpm container
See:
![image](https://user-images.githubusercontent.com/4009928/147275179-5dd3b9bd-0529-4cde-a270-539f810c6216.png)

**Comment by pabamato** [comment]
From Docker release notes:
https://docs.docker.com/desktop/mac/release-notes/3.x/#known-issues

> On Apple Silicon in native arm64 containers, older versions of libssl in debian:buster, ubuntu:20.04 and centos:8 will segfault when connected to some TLS servers, for example curl https://dl.yarnpkg.com. The bug is fixed in newer versions of libssl in debian:bullseye, ubuntu:21.04 and fedora:35.

**Comment by texorama** [comment]
Yes I think you do. We migrated the image to Ubuntu 21 and the ssl segfault has gone away.
